### PR TITLE
ci: restrict presubmit samples ITs to only snapshot

### PR DESCRIPTION
**Pull request by stephaniewang526**
to address incessant quota issues

### CODE REVIEW

File: .kokoro/build.sh [hunk 1]
```diff
@@ -69,14 +69,22 @@ integration)
...
         then
             source "${KOKORO_GFILE_DIR}/secret_manager/java-bigquerydatatransfer-samples-secrets"
         fi
 
-        pushd samples
+        # only run ITs in snapshot/ on presubmit PRs
+        if [[ ! -z ${KOKORO_GITHUB_PULL_REQUEST_NUMBER} ]]
+        then
+          SAMPLES_DIR=samples/snapshot
+        fi
```
Comment (chingor13) [inline .kokoro/build.sh:84]: Let's move this up by the first declaration of SAMPLES_DIR

File: .kokoro/build.sh [hunk 2]
```diff
@@ -69,14 +69,22 @@ integration)
     RETURN_CODE=$?
     ;;
 samples)
-    if [[ -f samples/pom.xml ]]
+    SAMPLES_DIR=samples
+    if [[ -f SAMPLES_DIR/pom.xml ]]
```
Comment (chingor13) [inline .kokoro/build.sh:73]:
```suggestion
    if [[ -f ${SAMPLES_DIR}/pom.xml ]]
```

File: .kokoro/build.sh [hunk 3]
```diff
@@ -69,14 +69,22 @@ integration)
...
 
-        pushd samples
+        # only run ITs in snapshot/ on presubmit PRs
+        if [[ ! -z ${KOKORO_GITHUB_PULL_REQUEST_NUMBER} ]]
+        then
+          SAMPLES_DIR=samples/snapshot
+        fi
+
+        # run ITs in all three samples/ subdirectories nightly
+        pushd SAMPLES_DIR/
```
Comment (chingor13) [inline .kokoro/build.sh:87]:
```suggestion
        pushd ${SAMPLES_DIR}
```

**Review by chingor13 — CHANGES_REQUESTED** [review]

File: .kokoro/build.sh (hunk 1 above)
Comment (stephaniewang526) [reply .kokoro/build.sh]: done

**Review by chingor13 — APPROVED** [review]

**Pull request merged**